apps/server/src/modules/user/service/user.service.spec.ts

```diff
@@ -123,6 +141,30 @@ describe('UserService', () => {
 		});
 	});
 
+	describe('findById', () => {
+		beforeEach(() => {
+			const userDO: UserDO = new UserDO({
+				firstName: 'firstName',
+				lastName: 'lastName',
+				email: 'email',
+				schoolId: 'schoolId',
+				roleIds: ['roleId'],
+				externalId: 'externalUserId',
+			});
+			userDORepo.findById.mockResolvedValue(userDO);
+		});
+
+		it('should provide information about the passed userId', async () => {
+			// Act
```

Comment:
no need for these comments, empty lines between sections are sufficient

Reply:
removed it